core/src/main/java/cz/xtf/core/openshift/OpenShiftBinaryManager.java

```diff
@@ -56,9 +59,13 @@ private OpenShiftBinary getBinary(String token, String username, String password
         } else {
             // If we are using an existing kubeconfig (or a default kubeconfig), we copy the original kubeconfig
             if (StringUtils.isNotEmpty(kubeconfig)) {
-                // We copy the specified kubeconfig
+                // flatten kubeconfig in case it contains certs/keys
                 try {
-                    Files.copy(Paths.get(kubeconfig), Paths.get(ocConfigPath), StandardCopyOption.REPLACE_EXISTING);
+                    Files.deleteIfExists(Paths.get(ocConfigPath));
+                    Files.write(Paths.get(ocConfigPath),
+                            Arrays.asList(new OpenShiftBinary(OpenShifts.getBinaryPath(), null)
+                                    .execute("config", "view", "--kubeconfig", kubeconfig, "--flatten")),
+                            StandardCharsets.UTF_8, StandardOpenOption.WRITE, StandardOpenOption.CREATE_NEW);
```

Comment:
can we flatten also default config bellow? line 77

Reply:
done